packages/client/auth/src/CrossmintAuthClient.ts

```diff
@@ -53,16 +62,22 @@ export class CrossmintAuthClient extends CrossmintAuth {
         this.callbacks.onLogout?.();
     }
 
-    public async handleRefreshToken(refreshTokenSecret?: string): Promise<void> {
+    public async handleRefreshAuthMaterial(refreshTokenSecret?: string): Promise<void> {
```

Comment:
Nit: I feel this name change isn't that big of a deal and it made you rename all the references everywhere.
Is there anything I'm missing here?

Reply:
I thought the original name was really bad, and as it hasn't shipped yet, it's not a breaking change. I think the new name is more consistent with the naming elsewhere